### PR TITLE
Kubernetes Resources

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds Kubernetes Resources docs to existing Live Containers docs

### Motivation
Jira request / product launch

### Preview link
https://docs-staging.datadoghq.com/sarina/k8s-resources/infrastructure/livecontainers/

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/infrastructure/livecontainers.md [hunk 1]
```diff
@@ -21,53 +21,58 @@ Taking inspiration from bedrock tools like *htop*, *ctop*, and *kubectl*, live c
...
+### Include/Exclude containers
 
-[1]: /agent/docker/log/?tab=containerinstallation
-[2]: /logs/indexes/#exclusion-filters
-{{% /tab %}}
+It is possible to include and/or exclude containers from real-time collection:
 
-{{% tab "Kubernetes" %}}
-In the `dd-agent.yaml` manifest used to create the [DaemonSet][1], add the following environment variables, volume mount, and volume:
+* Exclude containers either via passing the environment variable `DD_CONTAINER_EXCLUDE` or adding `container_exclude:` in your `datadog.yaml` main configuration file.
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:70]:
```suggestion
* Exclude containers either by passing the environment variable `DD_CONTAINER_EXCLUDE` or adding `container_exclude:` in your `datadog.yaml` main configuration file.
```

File: content/en/infrastructure/livecontainers.md [hunk 2]
```diff
@@ -21,53 +21,58 @@ Taking inspiration from bedrock tools like *htop*, *ctop*, and *kubectl*, live c
...
+* Include containers either via passing the environment variable `DD_CONTAINER_INCLUDE` or adding `container_include:` in your `datadog.yaml` main configuration file.
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:71]:
```suggestion
* Include containers either by passing the environment variable `DD_CONTAINER_INCLUDE` or by adding `container_include:` in your `datadog.yaml` main configuration file.
```

File: content/en/infrastructure/livecontainers.md [hunk 3]
```diff
@@ -84,45 +89,25 @@ volumes:
...
-{{< img src="infrastructure/livecontainers/errorlogs.png" alt="Preview Logs Sidepanel"  style="width:100%;">}}
+Navigate to the [Containers page][1]. This will automatically bring you to the **Containers** view.
 
 ## Searching, Filtering, and Pivoting
 
 ### String Search
 
 Containers are, by their nature, extremely high cardinality objects. Datadog's flexible string search matches substrings in the container name, ID, or image fields.
 
+If you've enabled Kubernetes Resources, strings such as Pod, deployment, ReplicaSet, and service name and Kubernetes labels are searchable in a [Kubernetes Resources view](#kubernetes-resources-views).
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:109]:
```suggestion
If you've enabled Kubernetes Resources, strings such as `pod`, `deployment`, `ReplicaSet`, and `service name`, as well as Kubernetes labels are searchable in a [Kubernetes Resources view](#kubernetes-resources-views).
```

File: content/en/infrastructure/livecontainers.md [hunk 4]
```diff
@@ -134,9 +119,25 @@ To combine multiple string searches into a complex query, you can use any of the
 
 Use parentheses to group operators together. For example, `(NOT (elasticsearch OR kafka) java) OR python`.
 
-### Tagging
+### Filtering and Pivoting
+
+The screenshot below displays a system that has been filtered down to a Kubernetes cluster of 9 nodes. RSS and CPU utilization on containers is reported compared to the provisioned limits on the containers, when they exist. Here, it is apparent that the containers in this cluster are over-provisioned. You could use tighter limits and bin packing to achieve better utilization of resources.
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:124]:
```suggestion
The screenshot below displays a system that has been filtered down to a Kubernetes cluster of nine nodes. RSS and CPU utilization on containers is reported compared to the provisioned limits on the containers, when they exist. Here, it is apparent that the containers in this cluster are over-provisioned. You could use tighter limits and bin packing to achieve better utilization of resources.
```

File: content/en/infrastructure/livecontainers.md [hunk 5]
```diff
@@ -134,9 +119,25 @@ To combine multiple string searches into a complex query, you can use any of the
...
+
+{{< img src="infrastructure/livecontainers/overprovisioned.png" alt="Over Provisioned"  style="width:80%;">}}
+
+Container environments are dynamic and can be hard to follow. The following screenshot displays a view that has been pivotted by `kube_service` and `host`—and, to reduce system noise, filtered to `kube_namespace:default`. You can see what services are running where, and how saturated key metrics are:
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:128]:
```suggestion
Container environments are dynamic and can be hard to follow. The following screenshot displays a view that has been pivoted by `kube_service` and `host`—and, to reduce system noise, filtered to `kube_namespace:default`. You can see what services are running where, and how saturated key metrics are:
```

File: content/en/infrastructure/livecontainers.md [hunk 6]
```diff
@@ -193,34 +186,65 @@ The query at the top of the scatter plot analytic allows you to control your sca
...
+
+For a detailed dashboard of this resource, click the View Pods Dashboard in the top right corner of this panel.
+
+### Container Logs
+
+View streaming logs for any container like `docker logs -f` or `kubectl logs -f` in Datadog. Click any container in the table to inspect it. Click the *Logs* tab to see real-time data from [Live Tail][18] or indexed logs for any time in the past.
+
+#### Live Tail
+
+With Live Tail, all container logs are streamed -- pausing the stream allows you to easily read logs that are quickly being written; un-pause to continue streaming.
```
Comment (cswatt) [inline content/en/infrastructure/livecontainers.md:223]:
```suggestion
With Live Tail, all container logs are streamed. Pausing the stream allows you to easily read logs that are quickly being written; unpause to continue streaming.
```

**Comment by sarina-dd** [comment]
Couldn't for the life of me figure out why the PHP file was showing - it looks like a link was removed from the file, but never re-added. [16] doesn't exist anymore. Tried removing the file, re-adding from master, unstaging, etc. but it keeps auto-generating. I'll fix the file within this PR so we can resolve the issue.

File: content/en/infrastructure/livecontainers.md [hunk 7]
```diff
@@ -21,53 +21,58 @@ Taking inspiration from bedrock tools like *htop*, *ctop*, and *kubectl*, live c
...
+   ...
+   agents:
+      image:
+        repository: datadog/agent
+        tag: 7.21.1
+        pullPolicy: Always
+   ...
+   clusterAgent:
+	repository: datadog/cluster-agent
+tag: 1.7.0
```
Comment (nammn) [inline content/en/infrastructure/livecontainers.md:59]: just to doublecheck. If this corresponds to the DCA version for orchestrator explorer, it should be `1.8.0`

File: content/en/infrastructure/livecontainers.md [hunk 8]
```diff
@@ -229,9 +253,19 @@ container_include: ["name:frontend.*"]
...
+[11]: /getting_started/tagging/
+[12]: /getting_started/tagging/unified_service_tagging
+[13]: /dashboards/widgets/timeseries/
+[14]: /logs
+[15]: /metrics
+[16]: /tracing
+[17]: /events
+[18]: /logs/live_tail/
+[19]: https://gist.github.com/hkaj/404385619e5908f16ea3134218648237
+[20]: https://github.com/DataDog/helm-charts
```
Comment (nammn) [inline content/en/infrastructure/livecontainers.md:271]: maybe use this link instead https://github.com/DataDog/helm-charts/tree/master/charts/datadog for a direct link to the configurable values

File: content/en/infrastructure/livecontainers.md (hunk 7 above)
Comment (cohenyair) [reply content/en/infrastructure/livecontainers.md:59]: actually both tags should be `tag: latest` now, and we specify the correct version in the requirements above (1.8.0 indeed)

Comment (cohenyair) [reply content/en/infrastructure/livecontainers.md:59]: we also have an example for the correct values in https://docs.google.com/document/d/1LiIvmynJ4hPsVytWSZXM3X3PZfXhVW9j0zcvVb3pi60/edit#

Comment (cohenyair) [reply content/en/infrastructure/livecontainers.md:59]: also note that lines 58-60 are not indented correctly
